.gitignore

```diff
@@ -32,3 +21,7 @@ Pods/
 Podfile.lock
 LumberjackUser*.h
 .DS_Store
+
+# SwiftPM
+.swiftpm
+.build
```

Comment:
NitPickk: Missing newline at end of file

Reply:
@ffried Is there any option for DangerFile which would check newlines at end of file?

Reply:
@ffried I am asking, because I know that you would like to separate it into another PR :)

Reply:
I think Danger could do this, yea. And yea, you're right, this is scope of another PR. 🙂 

Since you get the changed files, you could probably walk through them and check if they have a newline at the end of the document.

Reply:
@ffried 
Or I can setup local DangerFile with newline checks and ask it to do that. ;)